recipes/act-atmos/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "act-atmos" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c7cf956c516c736f0d5ea50fd81e41f88de3d6ce40803ef05cbd61b4fc8b541d
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pyproj >=2.3.1
+    - pandas
+    - xarray
+    - matplotlib
+    - numpy
+    - scipy
+    - astral
+    - dask
+    - distributed
+    - pint
+    - cartopy
+    - metpy
+    - boto3
+    - netcdf4
+
+test:
+  requires:
+    - pytest
+  imports:
+    - act
+  commands:
+    - py.test --pyargs act
+
+about:
+  home: https://github.com/ANL-DIGR/ACT
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Python library for working with atmospheric time-series by n-dimension datasets'
+
+  description: |
+    Toolkit for working with atmospheric data in every aspect
+    of the scientific process.  From discovery and downloading,
+    to visualization, to retrievals and advanced calculations
+  doc_url: https://anl-digr.github.io/ACT/
+  dev_url: https://anl-digr.github.io/ACT/
+
+extra:
+  recipe-maintainers:
+    - AdamTheisen
+    - rcjackson
+    - zssherman
```

Comment:
@rcjackson @zssherman are you OK with being maintainers?

Reply:
Hi @scopatz , thanks for taking a look at this, and I'm cool with being a maintainer.

Reply:
@scopatz Yes, I am ok with being a maintainer.

Reply:
Thanks!